modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -36,10 +38,37 @@ metadata:
 spec:
   sso:
     provider: keycloak
+    keycloak:
+      rootCA: "<PEM-encoded-root-certificate>" <1>
   server:
     route:
-     enabled: true
+      enabled: true
+----
+<1> A custom certificate used to verify the Keycloak's TLS certificate.
++ 
+The Operator reconciles changes in the `.spec.keycloak.rootCA` field and updates the `oidc.config` in `argocd-cm` configuration map with the PEM encoded root certificate.
```

Comment:
Refer to this guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

Consult with the SME and find out what object is `oidc.config` and accordingly write: "... the `oidc.config` <object>"